spec/factories/messages.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :message do
+    content "Hello, Welcome to the app!!"
+    sender_id 101
```

Comment:
Instead of hard coded ids use `Association` here to create a real object.
Because in this case if I create a `Message` using your factory say
```
m = create(:message)
```
and i want `m.sender` or `m.reciever`
I won't get them as we will not be having User with ids `101` and `102` in database.

Reply:
Yes sir, We merged user and message model locally and understood this problem. We are working on it.

Reply:
Okay coool